distribution/docs/src/main/jdocs/content/_plugins/catalog-metrics-plugin-contents.adoc

```diff
@@ -0,0 +1,17 @@
+:type: plugin
+:status: published
+:title: Catalog Metrics Plugin
+:link: _catalog_metrics_plugin
+:plugintypes: prequery
```

Comment:
❗️ This plugin implements PostIngestPlugin, PostQueryPlugin, PostResourcePlugin and PreQueryPlugin.